src/openrct2-ui/windows/GameBottomToolbar.cpp

```diff
@@ -614,17 +614,21 @@ static void window_game_bottom_toolbar_draw_news_item(rct_drawpixelinfo* dpi, rc
             auto image_id = ImageId(image_id_base, peep->TshirtColour, peep->TrousersColour);
             gfx_draw_sprite(&cliped_dpi, image_id, clipCoords);
 
-            if (image_id_base >= 0x2A1D && image_id_base < 0x2A3D)
-            {
-                gfx_draw_sprite(&cliped_dpi, ImageId(image_id_base + 32, peep->BalloonColour), clipCoords);
-            }
-            else if (image_id_base >= 0x2BBD && image_id_base < 0x2BDD)
-            {
-                gfx_draw_sprite(&cliped_dpi, ImageId(image_id_base + 32, peep->UmbrellaColour), clipCoords);
-            }
-            else if (image_id_base >= 0x29DD && image_id_base < 0x29FD)
+            auto* guest = peep->As<Guest>();
+            if (guest != nullptr)
```

Comment:
This would be a better use case:

```suggestion
            if (auto* guest = peep->As<Guest>(); guest != nullptr)
```

Reply:
We could even remove the `guest != nullptr` part, though I think our warning settings don't like implicit nullptr checks.